sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_analyze_conversation_app_async.py

```diff
@@ -30,47 +32,69 @@ async def sample_analyze_conversation_app_async():
+    client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
     async with client:
-        result = await client.analyze_conversations(
-            input,
-            project_name=conv_project,
-            deployment_name='production'
+        query = "Send an email to Carol about the tomorrow's demo"
+        result = await client.analyze_conversation(
+            task={
+                "kind": "CustomConversation",
+                "analysisInput": {
+                    "conversationItem": {
+                        "participantId": "1",
+                        "id": "1",
+                        "modality": "text",
+                        "language": "en",
+                        "text": query
+                    },
+                    "isLoggingEnabled": False
+                },
+                "parameters": {
+                    "projectName": project_name,
+                    "deploymentName": deployment_name,
+                    "verbose": True
+                }
+            }
         )
 
         # view result
-        print("query: {}".format(result.query))
-        print("project kind: {}\n".format(result.prediction.project_kind))
-
-        print("view top intent:")
-        print("\ttop intent: {}".format(result.prediction.top_intent))
-        print("\tcategory: {}".format(result.prediction.intents[0].category))
-        print("\tconfidence score: {}\n".format(result.prediction.intents[0].confidence_score))
-
-        print("view entities:")
-        for entity in result.prediction.entities:
-            print("\tcategory: {}".format(entity.category))
-            print("\ttext: {}".format(entity.text))
-            print("\tconfidence score: {}".format(entity.confidence_score))
+        print("query: {}".format(result.results.query))
+        print("project kind: {}\n".format(result.results.prediction.project_kind))
+
+        print("top intent: {}".format(result.results.prediction.top_intent))
+        print("category: {}".format(result.results.prediction.intents[0].category))
+        print("confidence score: {}\n".format(result.results.prediction.intents[0].confidence))
+
+        print("entities:")
+        for entity in result.results.prediction.entities:
+            print("\ncategory: {}".format(entity.category))
+            print("text: {}".format(entity.text))
+            print("confidence score: {}".format(entity.confidence))
+            if entity.resolutions:
+                print("resolutions")
+                for resolution in entity.resolutions:
+                    print("kind: {}".format(resolution.resolution_kind))
+                    print("value: {}".format(resolution.additional_properties["value"]))
+            if entity.extra_information:
+                print("extra info")
+                for data in entity.extra_information:
+                    print("kind: {}".format(data.extra_information_kind))
+                    print("value: {}".format(data.value))
```

Comment:
I think this is actually incorrect... if kind == 'ListKey', then it would be `data.key`.

Reply:
fixed